ApiBundle/Controller/TranslateController.php

```diff
@@ -26,10 +26,14 @@ class TranslateController extends BaseController
     public function translateAction(Request $request)
     {
         $displayedElements = $request->get('displayedElements');
+
+        $entityType = $this->container->camelize($request->get('entityType'));
+        $entityType = $this->container->underscore($entityType);
```

Comment:
I've got three kinds of entityType values : Camelize, underscore, lower case. If i underscore an underscored string, the method replace '_' by '.'. If I camelize a camelized string, there is no effect.